hw/bsp/hifive1/riscv_openocd.cfg

```diff
@@ -0,0 +1,34 @@
+adapter_khz     10000
```

Comment:
I wish it was my doing. Script is mostly taken from freedom-e-sdk.
I modified it to use same script for upload and debugging, part of sdk script was moved to bash script and later passed to already existing openocd.sh.